ironfish/src/wallet/account.ts

```diff
@@ -423,6 +425,11 @@ export class Account {
         tx,
       )) {
         if (!note.spent) {
+          if (notOnChainNotes.includes(note)) {
```

Comment:
This won't fix the bug, but it might cover it up.

It shouldn't be possible for a note to be in _both_ `loadNotesNotOnChain` and `loadNotesInSequenceRange`: if it's not on chain then it doesn't have a sequence, and if it has a sequence then it is on chain.